libs/platform/src/lib/components/form/time-picker/time-picker.component.html

```diff
@@ -0,0 +1,25 @@
+<fd-time-picker
+    [attr.aria-label]="ariaLabel"
+    [attr.ariaLabelledBy]="ariaLabelledBy"
+    [attr.ariaDescribedBy]="ariaDescribedBy"
+    [time]="time"
+    [compact]="contentDensity === 'compact'"
+    [meridian]="meridian"
+    [disabled]="disabled"
+    [spinnerButtons]="spinnerButtons"
+    [displaySeconds]="displaySeconds"
+    [displayMinutes]="displayMinutes"
+    [displayHours]="displayHours"
+    [placeholder]="placeholder"
+    [timePickerInputLabel]="timePickerInputLabel"
+    [allowNull]="allowNull"
+    [tablet]="tablet"
+    [placement]="placement"
+    [state]="state"
+    [buttonFocusable]="buttonFocusable"
+    [keepTwoDigitsTime]="keepTwoDigitsTime"
+    (isOpenChange)="handleOpenChange($event)"
+    (ngModelChange)="handleTimeChange($event)"
+    [(ngModel)]="value">
```

Comment:
I'm wondering if it's valid to keep `[time]="time"` model along with the `[(ngModel)]="value"`.
What is the source of truth in this case?

Reply:
@dimamarksman, I don't know about a strong source of truth, I saw a similar implementation in platform datepicker.

In my opinion, it's a good way to provide all available options to manage the control, and it's a common practice to have an `ngModel` on the "input"/form element.

In any case, I think it's better to discuss this point with the platform team directly.

Reply:
I meant that we probably need to control underlying core-time-picker using only one way - or [ngModel] or [time]

Reply:
Probably not a big deal, just found it a little bit confusing